Sources/ApodiniUtils/Swift.swift

```diff
@@ -55,6 +55,11 @@ extension Array {
         copy.append(contentsOf: elements)
         return copy
     }
+    
+    /// Sorts the array in-place, using a key path
+    public mutating func sort<T: Comparable>(by keyPath: KeyPath<Element, T>) {
```

Comment:
The difference between this one and the one you linked is that mine takes a KeyPath, whereas the one in the stdlib takes a closure. (So you could do `names.sort(\.count)`, rather than `names.sort { $0.count < $1.count }`)

Reply:
Yes, I know. The non-mutating KeyPath based on is a custom extension located a few lines below. Would have envisioned a similar version for the mutating one.
I would have envisioned that doing a mutating sort may be less expensive than doing a memory copy and sorting on that. But haven't done any measurements.

Reply:
Oh yeah I'll probably be able to fix that by simply implementing them all in terms of the mutating one in the stdlib. I'll see if i can get that working